### PR TITLE
Use mamba instead of conda where possible

### PR DESCRIPTION
**Pull request by mathbunnyru**

### CODE REVIEW

File: docs/using/common.md [hunk 1]
```diff
@@ -161,35 +161,35 @@ You can use either `pip`, `conda` or `mamba` to install new packages without any
...
-However, alternative channels can be used either one shot by overwriting the default channel in the installation command or by configuring `conda` to use different channels.
+However, alternative channels can be used either one shot by overwriting the default channel in the installation command or by configuring `mamba` to use different channels.
 The examples below show how to use the [anaconda default channels](https://repo.anaconda.com/pkgs/main) instead of `conda-forge` to install packages.
 
 ```bash
 # using defaults channels to install a package
-conda install --channel defaults humanize
+mamba install --channel defaults humanize
 # configure conda to add default channels at the top of the list
 conda config --system --prepend channels defaults
```
Comment (mathbunnyru) [inline docs/using/common.md:189]: Currently, only install, create, list, search, run, info and clean are supported through mamba.

File: docs/using/recipes.md [hunk 2]
```diff
@@ -521,29 +521,6 @@ RUN pip install --quiet --no-cache-dir jupyter_contrib_nbextensions && \
 
 Ref: <https://github.com/jupyter/docker-stacks/issues/675>
 
-## Enable auto-sklearn notebooks
```
Comment (mathbunnyru) [inline docs/using/recipes.md:524]: This is easily installable as `mamba install auto-sklearn`, so let's remove this old instruction.

**Comment by mathbunnyru** [comment]
I didn't change env variables like CONDA_DIR and so on to save compatibility.
Ideally, this change is 100% compatible.

**Comment by consideRatio** [comment]
@mathbunnyru in this PR I felt you managed to comment exactly the kind of info I needed to feel at ease about the changes :) Thanks for making it so easy for me as a reviewer, and thank you for your thorough work!

**Review by consideRatio — APPROVED** [review]

**Pull request merged**